README.md

```diff
@@ -1,14 +1,14 @@
 dcafs
 =========
 
-A Java tool (or library) that takes care of all the nitty-gritty that needs to be done when a sensor generated data and you want to find that data in a database. Hence _data collect alter forward store_ this is in broad terms what it is capable of.
+A Java tool (or can also be used as a library) that takes care of all the nitty-gritty that needs to be done when a sensor has generated data and you want to find that data in a database. Hence _data collect alter forward store_. This is in broad terms what it is capable of.
 
 
 ## Main features
 * Collect data from TCP, UDP, MQTT, Websocket, serial/tty, I2C, SPI, email
 * Alter with string and math operations, or filter lines out
-* Forward back to origin, any other source, hosted tcp server or email eg. create a serial to tcp converter
-* Store processed data in SQLite, MariaDB, MySQL, InfluxDB and MSSQL while raw is stored in timestamped .log files
+* Forward back to origin, to any other source, a hosted tcp server or email (eg. even create a serial to tcp converter)
+* Store processed data in SQLite, MariaDB, MySQL, InfluxDB and MSSQL (dcafs will create these for you!) while additionally raw can be kept in timestamped .log files
```

Comment:
Not correct, sqlite is generated others need to be made (tables are work in progress for those)